2019-12-18 11:05:51,789 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-18 11:06:43,639 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-18 11:06:43,640 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-18 11:06:43,641 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-18 11:06:43,643 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:17:43,242 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:17:43,243 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:17:43,244 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:17:43,248 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:18:07,832 ERROR: Exception on /register [POST] [in /Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py:1890]
Traceback (most recent call last):
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/timgilboy/Downloads/recipebox/venv/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/users/timgilboy/Downloads/recipebox/recipe-box/app/main/routes.py", line 241, in register
    password_hash = generate_password_hash(form.password.password)
AttributeError: 'PasswordField' object has no attribute 'password'
2019-12-25 14:18:39,146 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:18:39,154 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:18:39,168 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
2019-12-25 14:18:39,216 INFO: Recipe book startup [in /users/timgilboy/Downloads/recipebox/recipe-box/app/__init__.py:77]
